HoloPipelines/pipelines/pipelineNifti2obj.py

```diff
@@ -0,0 +1,17 @@
+# this pipeline may be removed in the future as obj is not used to display 3D model on hololens
+from components import compNifti2numpy
+from components import compNumpy2obj
+import pathlib
+import sys
+
+
+def main(inputNiftiPath, outputObjPath, threshold, flipNpy=False):
+    generatedNumpyList = compNifti2numpy.main(str(pathlib.Path(inputNiftiPath)))
+    generatedObjPath = compNumpy2obj.main(
```

Comment:
same here. make use of this.

Reply:
output path variable now used in final print of a pipeline